### PR TITLE
I Fix The Mingun Osprey Through Sheer Willpower And Luck

### PR DESCRIPTION
**Pull request by Readystorm546**
So a long time ago i made this gun and then it was bad so i stopped making it, Then it somehow got merged so i took it as a sign and decided to try and "fix it" And i did. Also i did some more balance to the best of my ability.

Here i shoot a window
| 
|
V

ezgif com-video-to-gif

Here i shoot innocent civilians at close range
|
|
V

ezgif com-video-to-gif (1)

Here i shoot innocent civilians at long range
|
|
V

ezgif com-video-to-gif (2)

Here i shoot innocent doors at long range
|
|
V

ezgif com-video-to-gif (3)

And this is the text it reads when you are shooting and when you have shot too much
|
|
V

ezgif com-video-to-gif (4)

Better leave both hands free to fire...

dreamseeker_3TUUWooB5E

Stats for the gun include

	slowdown = 2 // The Slowdown effect on the player while equipped
	w_class = WEIGHT_CLASS_HUGE // Its big woah
	fire_delay = 1 // Time between each bullet in a  burst
	burst_size = 3 / /How many bullets per click
        automatic = 0 // Cool-down between bursts 
	recoil = 0.5 // Screen shake factor
	spread = 34 // The way the bullets come out
	fire_sound_volume = 60 //How loud the gun is
	overheat = 0 // Starting overheat level
	overheat_max = 30 // How far the overheat can go
	heat_diffusion = 2 // How fast heat diffuses 
        damage = 34 // How much damage each bullet does (Brute)
        TC = 36 // How much TC it costs (Only available in nuke opps)

Dont try and hold down to shoot the gun anymore. Just click to shoot a volley of 3 and do it fast fast fast.

#### Changelog

:cl:    
bugfix: I mean i just fixed the entire gun to work well
tweak: Made some balance changes to the gun  
/:cl:

### CODE REVIEW

**Comment by monster860** [comment]
[Cry some more](https://wiki.teamfortress.com/w/images/6/65/Heavy_specialcompleted07.wav)

**Comment by Hopekz** [comment]
> [Cry some more](https://wiki.teamfortress.com/w/images/6/65/Heavy_specialcompleted07.wav)

It costs 400,000 credits to fire this weapon....for 120 deciseconds

**Comment by Nickvr628** [comment]
That small burst and delay doesn't feel minigun is... maybe a 7-10 round burst would be better?

**Comment by Readystorm546** [comment]
> That small burst and delay doesn't feel minigun is... maybe a 7-10 round burst would be better?

I was thinking about this too. But i think i have another solution in which it just fires faster because you can end up spamming it.

File: yogstation/code/modules/projectiles/guns/ballistic/minigun.dm [hunk 1]
```diff
@@ -100,27 +101,23 @@
 	lefthand_file = 'yogstation/icons/mob/inhands/weapons/minigun_inhand_left.dmi'
 	righthand_file = 'yogstation/icons/mob/inhands/weapons/minigun_inhand_right.dmi'
 	flags_1 = CONDUCT_1
-	slowdown = 1.6
+	slowdown = 2
 	slot_flags = null
 	w_class = WEIGHT_CLASS_HUGE
 	materials = list()
+	burst_size = 3
+	automatic = 0
```
Comment (alexkar598) [inline yogstation/code/modules/projectiles/guns/ballistic/minigun.dm]:
```suggestion
	automatic = FALSE
```

**Review by alexkar598 — CHANGES_REQUESTED** [review]

**Review by alexkar598 — APPROVED** [review]

**Pull request merged**